### ISSUE TITLE
Error: Could not locate the bindings file.

### ISSUE THREAD

**Issue by 3amr7ussein** [ibmdb/node-ibm_db#909]
D:\Porjects\test\node_modules\bindings\bindings.js:126
  err = new Error(
        ^
Error: Could not locate the bindings file. Tried:
 → D:\Porjects\test\node_modules\ibm_db\build\odbc_bindings.node
 → D:\Porjects\test\node_modules\ibm_db\build\Debug\odbc_bindings.node
 → D:\Porjects\test\node_modules\ibm_db\build\Release\odbc_bindings.node
 → D:\Porjects\test\node_modules\ibm_db\out\Debug\odbc_bindings.node
 → D:\Porjects\test\node_modules\ibm_db\Debug\odbc_bindings.node
 → D:\Porjects\test\node_modules\ibm_db\out\Release\odbc_bindings.node
 → D:\Porjects\test\node_modules\ibm_db\Release\odbc_bindings.node
 → D:\Porjects\test\node_modules\ibm_db\build\default\odbc_bindings.node
 → D:\Porjects\test\node_modules\ibm_db\compiled\18.14.2\win32\x64\odbc_bindings.node
 → D:\Porjects\test\node_modules\ibm_db\addon-build\release\install-root\odbc_bindings.node
 → D:\Porjects\test\node_modules\ibm_db\addon-build\debug\install-root\odbc_bindings.node
 → D:\Porjects\test\node_modules\ibm_db\addon-build\default\install-root\odbc_bindings.node
 → D:\Porjects\test\node_modules\ibm_db\lib\binding\node-v108-win32-x64\odbc_bindings.node
    at bindings (D:\Porjects\test\node_modules\bindings\bindings.js:126:9)
    at Object.<anonymous> (D:\Porjects\test\node_modules\ibm_db\lib\odbc.js:57:31)
    at Module._compile (node:internal/modules/cjs/loader:1254:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1308:10)
    at Object.require.extensions.<computed> [as .js] (D:\Porjects\test\node_modules\ts-node\src\index.ts:1608:43)
    at Module.load (node:internal/modules/cjs/loader:1117:32)
    at Function.Module._load (node:internal/modules/cjs/loader:958:12)
    at Module.require (node:internal/modules/cjs/loader:1141:19)
    at require (node:internal/modules/cjs/helpers:110:18)
    at Object.<anonymous> (D:\Porjects\test\src\database.ts:3:15) {
  tries: [
    'D:\\Porjects\\test\\node_modules\\ibm_db\\build\\odbc_bindings.node',
    'D:\\Porjects\\test\\node_modules\\ibm_db\\build\\Debug\\odbc_bindings.node',
    'D:\\Porjects\\test\\node_modules\\ibm_db\\build\\Release\\odbc_bindings.node',
    'D:\\Porjects\\test\\node_modules\\ibm_db\\out\\Debug\\odbc_bindings.node',
    'D:\\Porjects\\test\\node_modules\\ibm_db\\Debug\\odbc_bindings.node',
    'D:\\Porjects\\test\\node_modules\\ibm_db\\out\\Release\\odbc_bindings.node',
    'D:\\Porjects\\test\\node_modules\\ibm_db\\Release\\odbc_bindings.node',
    'D:\\Porjects\\test\\node_modules\\ibm_db\\build\\default\\odbc_bindings.node',
    'D:\\Porjects\\test\\node_modules\\ibm_db\\compiled\\18.14.2\\win32\\x64\\odbc_bindings.node',
    'D:\\Porjects\\test\\node_modules\\ibm_db\\addon-build\\release\\install-root\\odbc_bindings.node',
    'D:\\Porjects\\test\\node_modules\\ibm_db\\addon-build\\debug\\install-root\\odbc_bindings.node',
    'D:\\Porjects\\test\\node_modules\\ibm_db\\addon-build\\default\\install-root\\odbc_bindings.node',
    'D:\\Porjects\\test\\node_modules\\ibm_db\\lib\\binding\\node-v108-win32-x64\\odbc_bindings.node'
  ]
}
[nodemon] app crashed - waiting for file changes before starting...

**Comment by 3amr7ussein** [comment]
I've download the driver manually but now iam stuck with this error when i try to establish connection

D:\Porjects\test\node_modules\bindings\bindings.js:121
        throw e;
        ^

Error: The specified module could not be found.
\\?\D:\Porjects\test\node_modules\ibm_db\build\Release\odbc_bindings.node
    at Module._extensions..node (node:internal/modules/cjs/loader:1338:18)
    at Module.load (node:internal/modules/cjs/loader:1117:32)
    at Module._load (node:internal/modules/cjs/loader:958:12)
    at Module.require (node:internal/modules/cjs/loader:1141:19)
    at require (node:internal/modules/cjs/helpers:110:18)
    at bindings (D:\Porjects\test\node_modules\bindings\bindings.js:112:48)
    at Object.<anonymous> (D:\Porjects\test\node_modules\ibm_db\lib\odbc.js:57:31)
    at Module._compile (node:internal/modules/cjs/loader:1254:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1308:10)
    at Module.load (node:internal/modules/cjs/loader:1117:32) {
  code: 'ERR_DLOPEN_FAILED'
}

Node.js v18.14.2

**Comment by bimalkjha** [comment]
@3amr7ussein Please share complete output of `npm install ibm_db` command along with other info requested in new issue template. Thanks.

**Comment by 3amr7ussein** [comment]
[2023-03-12T06_31_11_878Z-debug-0.log](https://github.com/ibmdb/node-ibm_db/files/10950166/2023-03-12T06_31_11_878Z-debug-0.log)
Here it is it seems there is some errors at installation

**Comment by bimalkjha** [comment]
@3amr7ussein From windows command prompt, execute below commands:
```
cd D:\Porjects\test\node_modules\ibm_db
npm install
```
Now, share the complete output of `npm install` command and output of `dir D:\Porjects\test\node_modules\ibm_db\build\Release` command. If you see, the `npm install` command prints `ibm_db installed successfully!` at end, then try running your test program. Thanks.

**Comment by bimalkjha** [comment]
@3amr7ussein Please share requested info to make progress on this issue. Thanks.
